client/src/components/StatusPill/StatusPill.jsx

```diff
@@ -1,7 +1,7 @@
 import classNames from 'classnames'
 
 export function StatusPill({ value }) {
-  const status = value ? value.toLowerCase() : "unknown";
+  const status = value ? 'active' : 'inactive';
```

Comment:
Required because subscribers table uses a Boolean to store whether a subscriber is unsubscribed